src/base/bittorrent/session.cpp

```diff
@@ -381,7 +381,14 @@ Session::Session(QObject *parent)
     , m_diskCacheSize(BITTORRENT_SESSION_KEY(u"DiskCacheSize"_qs), -1)
     , m_diskCacheTTL(BITTORRENT_SESSION_KEY(u"DiskCacheTTL"_qs), 60)
     , m_diskQueueSize(BITTORRENT_SESSION_KEY(u"DiskQueueSize"_qs), (1024 * 1024))
+#if (LIBTORRENT_VERSION_NUM >= 20006)
+    , m_diskIOReadMode(BITTORRENT_SESSION_KEY(u"DiskIOReadMode"_qs), DiskIOReadMode::EnableOSCache
+        , clampValue(DiskIOReadMode::EnableOSCache, DiskIOReadMode::DisableOSCache))
+    , m_diskIOWriteMode(BITTORRENT_SESSION_KEY(u"DiskIOWriteMode"_qs), DiskIOWriteMode::WriteThrough
```

Comment:
I don't think using `WriteThrough` by default is a good idea.
The code is newly introduced, there might be unknown side-effects/bugs/etc..
I would suggest that the default is switched in later qbt version.